<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMountPointDialogWidgetBase</class>
 <widget class="QWidget" name="EditMountPointDialogWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>374</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="2">
    <widget class="QRadioButton" name="m_RadioUUID">
     <property name="text">
      <string>UU&amp;ID</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QSpinBox" name="m_SpinDumpFreq"/>
   </item>
   <item row="7" column="3" colspan="4">
    <widget class="QCheckBox" name="m_CheckUsers">
     <property name="text">
      <string>Users can mount and unmount</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3" colspan="4">
    <widget class="QCheckBox" name="m_CheckRelAtime">
     <property name="text">
      <string>Update access times relative to modification</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLabel" name="m_LabelTypeValue">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QCheckBox" name="m_CheckReadOnly">
     <property name="text">
      <string>Read-only</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="m_LabelPassNumber">
     <property name="text">
      <string>Pass &amp;Number:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_SpinPassNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QPushButton" name="m_ButtonMore">
     <property name="text">
      <string>More...</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3" colspan="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>195</width>
       <height>22</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QRadioButton" name="m_RadioDeviceNode">
     <property name="text">
      <string>De&amp;vice Node</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="m_LabelOptions">
     <property name="text">
      <string>Options:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3">
    <widget class="QRadioButton" name="m_RadioLabel">
     <property name="text">
      <string>&amp;Label</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="2">
    <widget class="QCheckBox" name="m_CheckNoExec">
     <property name="text">
      <string>No binary execution</string>
     </property>
    </widget>
   </item>
   <item row="12" column="4">
    <widget class="QSpinBox" name="m_SpinPassNumber"/>
   </item>
   <item row="12" column="5" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>135</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QCheckBox" name="m_CheckNoAuto">
     <property name="text">
      <string>No automatic mount</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QPushButton" name="m_ButtonRemove">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="m_ButtonSelect">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Select...</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3" colspan="4">
    <widget class="QCheckBox" name="m_CheckNoDirAtime">
     <property name="text">
      <string>No update of directory access times</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="m_LabelDumpFreq">
     <property name="text">
      <string>Dump &amp;Frequency:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>m_SpinDumpFreq</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="m_LabelType">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="m_LabelPath">
     <property name="text">
      <string>Path:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QLabel" name="m_LabelNameValue">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="7">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3" colspan="3">
    <widget class="QCheckBox" name="m_CheckNoAtime">
     <property name="text">
      <string>No update of file access times</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QCheckBox" name="m_CheckSync">
     <property name="text">
      <string>Synchronous access</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Identify by:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="3">
    <widget class="QComboBox" name="m_EditPath"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
